core/store/models/job_run.go

```diff
@@ -31,6 +31,7 @@ type JobRun struct {
 	CreationHeight *utils.Big   `json:"creationHeight"`
 	ObservedHeight *utils.Big   `json:"observedHeight"`
 	Overrides      JSON         `json:"overrides"`
+	InitialMeta    JSON         `json:"initialMeta" gorm:"default:'{}'"`
```

Comment:
Should the default be set on the database level?  Maybe it would be best to do when creating the Job Runs.